content/api/api-elements.md

```diff
@@ -7,205 +7,1684 @@ author: Lawrence Lane
+        "sum": 52,
+        "baselineStddev": 146.9486822676629,
+        "baselineUpper": 641.3675227576849,
+        "baselineLower": 0
+      }
+    }
+  ]
+}
+```
+
+{{% /expand %}}
 
+---
 
-**Parameters**
+## GET from /elements/{elementId}/metrics/{metricId}/tags
 
-| Parameters | Required/Optional | Description |
-|------------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| id | Optional | Query parameter. The FQN of the element(s). |
-| name | Optional | Query parameter. The name of the element. |
-| type | Optional | Query parameter. The type of element. |
-| location | Optional | Query parameter. The location of the element is used for filtering and display purposes in CloudWisdom. The location can reflect either the element’s physical location or business needs. |
-| tags | Optional | Query parameter. The name of the tag. |
-| startTime | Optional | Query parameter. The start of the window of time from which elements will be returned. The startTime must be in ISO 8601 format. The default startTime is 12:00 AM in the authenticating user’s specified time zone. |
-| endTime | Optional | Query parameter. The end of the window of time from which elements will be returned. The endTime must be in ISO 8601 format. The default endTime is the current time. |
-| state | Optional | Query parameter. The state of the element reflected by the netuitive.metrics.collected.percent metric. False means no metrics are being collected; True means any number of metrics are being collected. |
+{{< button theme="info" href="https://app.metricly.com/swagger-ui.html#!/elements/getMetricTagsUsingGET" >}} GET {{< /button >}} Use this endpoint to get a list of tags associated to a specified metric for a given element.
 
+{{% expand "View Method Details." %}}
 
-### GET Metric Metadata from /elements/{elementid}/metrics
-This method returns a list of metadata for a metric.
+### Parameters
 
-**Parameters**
+| Parameter | Parameter Type | Data Type | Description |
+|-----------|----------------|-----------|--------------------------|
+| elementId | Path | String | Element Ids can be found in by selecting an element in  CloudWisdom and viewing the URL (/#/inventory/6bdf4fd1-1111-1111-1c11-17eb5f628e46) |
+| metricId | Path | String | Metric Ids can be found by viewing metrics in CloudWisdom. |
 
-| Parameters | Required/Optional | Description |
-|------------|-------------------|-----------------------------------------------------|
-| elementId | Required | URL (path) parameter. The ID of the element. |
-| metricFQN | Optional | Query parameter. The metric’s fully qualified name. |
+### Request URL
 
-### GET Metric Results For an Element from /elements/{elementid}/metrics/{metricid}/samples
+```
+https://app.metricly.com/swagger-ui.html#!/elements/getMetricTagsUsingGET
```

Comment:
Should this be a swagger URL? The other request URLs look to now have swagger in the path, but were created within it.